Documentation/network/kubernetes/local-redirect-policy.rst

```diff
@@ -85,6 +85,65 @@ Validate that the Cilium Local Redirect Policy CRD has been registered.
 	   [...]
 	   ciliumlocalredirectpolicies.cilium.io              2020-08-24T05:31:47Z
 
+.. note::
+
+    Local Redirect Policy supports either the socket-level loadbalancer or the tc loadbalancer.
+    The configuration depends on your specific use case and the type of service handling required.
+    Below are the Helm setups to work with ``localRedirectPolicy=true``:
+
+    1. Enable full kube-proxy replacement:
+
+      - This setup is for users who want to replace kube-proxy with Cilium's eBPF implementation
```

Comment:
There is no list, so I'd remove the `-` and make it a regular paragraph. Same for other cases.